dashboard.el

```diff
@@ -118,6 +118,19 @@
   :type 'boolean
   :group 'dashboard)
 
+(defcustom dashboard-startupify-list
+  '((banner    . dashboard-insert-banner)
```

Comment:
I think it's a good idea to have this list, makes the dashboard more flexible.
 
We don't need the "name" part of the list, could be just the function.
For the new-line part, maybe we could define a `dashboard-insert-new-line` with an optional number of new lines to insert, and another `dashboard-insert-page-separator` that inserts the "page-separator"